desktop/windows/release-notes/index.md

```diff
@@ -17,6 +17,38 @@ redirect_from:
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
+## Docker Desktop 4.2.0
+2021-11-xx
+
+> Download Docker Desktop
+>
+> [For Windows](?: .button .primary-btn }
```

Comment:
Copy line 57 (from 4.1.1) to have the correct link for the latest release.
And then line 57 needs to be updated like line 69 without UTM, but with the build number for 4.1.1.